testgrid/config.yaml

```diff
@@ -4319,11 +4319,11 @@ dashboards:
     test_group_name: ci-kubernetes-e2e-gci-gce-reboot
   - name: gci-gce-scalability
     test_group_name: ci-kubernetes-e2e-gci-gce-scalability
-  - name: gci-gce-scalability-1.12
+  - name: gci-gce-scalability-1.13
```

Comment:
Any reason to not give them more stable names.. like stable1/stable2/..?

Reply:
So we don't need to keep changing them every release.

Reply:
we still want to keep the tab name explicit? aka you might not know what exactly is stable1...

Reply:
your choice :-)

Reply:
FWIW at present we use a script in `experiments` to touchup tab names based on the stable1/stable2/etc suffixes... it's done manually though, would be nice to autogenerate but that gets us into a pattern of either having the testgrid config-as-uploaded not live in test-infra, or having bots update the contents of PRs to testgrid config prior to merge, neither of which we've explored yet